docs/tutorials/google/xeb_calibration_example.ipynb

```diff
@@ -0,0 +1,674 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        calibration.heatmap(metric).plot(ax=axes[i] if len(metrics) &gt; 1 else axes);
</pre><p>If it's useful here, I think it might be worth showing off the heatmap feature which allows you to specify a different scaling (eg: LogNorm, PowerNorm etc.) for the mapping of colors. For Eg:</p><p> </p><p><code>heatmap.plot(ax, collection_options = {'norm' : mpl.colors.LogNorm()})</code>  could be useful to have better distinction among qubits with low error rates. This is also something which people use internally (eg: Wojtek had reached out to me asking for this feature). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Done!</p>